dockers/docker-orchagent/orchagent.sh

```diff
@@ -64,6 +64,9 @@ elif [ "$platform" == "mellanox" ]; then
     ORCHAGENT_ARGS+=""
 elif [ "$platform" == "innovium" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
+elif [ "$platform" == "pensando" ]; then
+    MAC_ADDRESS=$(ip link show oob_mnic0 | grep ether | awk '{print $2}')
```

Comment:
why cannot we rename oob_mnic0 to eth0?

Reply:
its named as oob_mnic0 across pensando fw in multiple places.

Reply:
git grep eth0

Reply:
We will check if we can alternate interface with name eth0 which points to oob_mnic0

Reply:
Renamed eth0 to oob_mnic0